api/google-search/src/main/java/com/google/search/googlesearch/services/SearchAutoCompleteService.java

```diff
@@ -0,0 +1,71 @@
+package com.google.search.googlesearch.services;
+
+import java.util.ArrayList;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SearchAutoCompleteService {
+
+	private ArrayList<String> keywordArray;
+
+	public ArrayList<String> matchedKeywords;
+
+
+	public SearchAutoCompleteService() {
+		super();
+		this.getKeywordArrayList();
```

Comment:
@ramsunvtech : AFAIK, it's not necessary that constructor should return something since it acts as a setter or for the instantiated object, I have used it for setting the data. 

A constructor is part of the Class lifecycle. I may be missing something in my knowledge. Keeping this comment open for rest of the team to comment on it.

Reply:
i'm ok for now 👍